<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>693</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ScanControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>491</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>开始扫描</string>
     </property>
     <property name="icon">
      <iconset resource="ico/ico.qrc">
       <normaloff>:/new/prefix1/video_play.png</normaloff>:/new/prefix1/video_play.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="settings">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>参数设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>调试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>加载文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="launchDevice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>启动激光</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>计算高差</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QSpinBox" name="check_num">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>测量区域数量:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="setPoint">
        <property name="text">
         <string>绘制测量点</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>测量阈值:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="limitValue">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>471</width>
       <height>224</height>
      </rect>
     </property>
     <property name="title">
      <string>测量结果</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <column>
         <property name="text">
          <string>结果</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起点</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>终点</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>最小值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>最大值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>高差</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="view_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px;
border-radius:25px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px;
border-radius:25px</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="base">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>491</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="threeDView"/>
     <widget class="QWidget" name="realTimeView">
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QWidget" name="singleFrameView"/>
     <widget class="QWidget" name="twoDView"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>481</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="imgLeft">
        <property name="toolTip">
         <string>左移</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/file/icons/ic_back.png</normaloff>:/file/icons/ic_back.png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgRight">
        <property name="toolTip">
         <string>右移</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/file/icons/ic_forward.png</normaloff>:/file/icons/ic_forward.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgUp">
        <property name="toolTip">
         <string>上移</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/ic_up.png</normaloff>:/icons/ic_up.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgDown">
        <property name="toolTip">
         <string>下移</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/ic_down.png</normaloff>:/icons/ic_down.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgBig">
        <property name="toolTip">
         <string>放大</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/file/icons/ic_zoom_in.png</normaloff>:/file/icons/ic_zoom_in.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgSmall">
        <property name="toolTip">
         <string>缩小</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/file/icons/ic_zoom_out.png</normaloff>:/file/icons/ic_zoom_out.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgRotate">
        <property name="toolTip">
         <string>旋转</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/file/icons/ic_undo.png</normaloff>:/file/icons/ic_undo.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="twoDButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2D显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeDButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3D显示</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="debug">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>461</width>
      <height>65</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="realTimeButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>实时显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="singleFrameButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单帧显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_Net_Param"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>任务(&amp;N)</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_start"/>
    <addaction name="action"/>
    <addaction name="action_pointAnalyze"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_S"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px;
border-radius:25px;
background-color: rgb(230, 230, 255);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_start"/>
   <addaction name="action_Quit"/>
  </widget>
  <action name="action_Net_Param">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/graph.png</normaloff>:/new/prefix1/graph.png</iconset>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/home.png</normaloff>:/new/prefix1/home.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_CameraLight_Param">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/email.png</normaloff>:/new/prefix1/email.png</iconset>
   </property>
   <property name="text">
    <string>相机和光源参数</string>
   </property>
  </action>
  <action name="action_Connection">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/chat_bubble.png</normaloff>:/new/prefix1/chat_bubble.png</iconset>
   </property>
   <property name="text">
    <string>连接(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>连接网络</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/folder.png</normaloff>:/new/prefix1/folder.png</iconset>
   </property>
   <property name="text">
    <string>加载文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>加载</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/cancel.png</normaloff>:/new/prefix1/cancel.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_imgset">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/user.png</normaloff>:/new/prefix1/user.png</iconset>
   </property>
   <property name="text">
    <string>图像检测参数</string>
   </property>
  </action>
  <action name="action_start">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/video_play.png</normaloff>
     <normalon>:/new/prefix1/minimize.png</normalon>:/new/prefix1/video_play.png</iconset>
   </property>
   <property name="text">
    <string>开始检测(&amp;K)</string>
   </property>
   <property name="toolTip">
    <string>开始检测/停止检测</string>
   </property>
  </action>
  <action name="action_big">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/add.png</normaloff>:/new/prefix1/add.png</iconset>
   </property>
   <property name="text">
    <string>放大(&amp;B)</string>
   </property>
  </action>
  <action name="action_small">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/minimize.png</normaloff>:/new/prefix1/minimize.png</iconset>
   </property>
   <property name="text">
    <string>缩小(&amp;S)</string>
   </property>
  </action>
  <action name="action_EXCEL_T">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/new/prefix1/email.png</normaloff>:/new/prefix1/email.png</iconset>
   </property>
   <property name="text">
    <string>导出EXCEL(T)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>网络测试</string>
   </property>
  </action>
  <action name="action_pointAnalyze">
   <property name="text">
    <string>点云分析</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
  <include location="ico/ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>action_start</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
